spec/unit/provider/package/windows/registry_uninstall_entry_spec.rb

```diff
@@ -22,4 +24,40 @@
       expect(quiet_uninstall_string_key).to eql "QuietUninstallStringPath"
     end
   end
+
+  describe "when pass 'nil' to find_entries", :windows_only do
+    let(:package_name) { nil }
+    it "should return empty entries array" do
```

Comment:
Minor changes: `it "returns empty array" do`. https://github.com/reachlocal/rspec-style-guide#should-it-or-should-not-in-it-statements